hack/testing/logging.sh

```diff
@@ -458,6 +461,29 @@ if [ ! -n "$USE_LOGGING_DEPLOYER_SCRIPT" ] ; then
 fi
 
 #upgrade-test
+function removeEsCuratorConfigMaps() {
```

Comment:
might want to see how I'm thinking of compressing tests in https://github.com/openshift/origin-aggregated-logging/pull/160